tuner/tuner/common.py

```diff
@@ -266,3 +268,41 @@ class MLIRTransformation:
     template: list[str]
     modified: str
     embeddable: str
+
+
+def combine_tuning_specs(
```

Comment:
Why `combine` instead of `merge`? I think we should stick with consistent naming if we can

Reply:
Could you explain what this function does as a comment so that it's easy to understand the difference between combine/link/merge?